src/docs/product/performance/web-vitals.mdx

```diff
@@ -71,6 +73,48 @@ Some Web Vitals such as FP, FCP, LCP, and TTFB are measured relative to the star
 
 </Note>
 
+## Web Vitals Page
+
+The **Web Vitals** page gives you an overview of your page load performance for the selected project(s). You can use this page as a starting point to investigate poor Web Vitals affecting your website and drill down to better understand which pages affect your website performance most.
```

Comment:
Question about "website performance" - would it be more relatable to say web app performance or website performance?

Reply:
Probably web application for consistency. Did a quick search in all the docs and it looks like we don't use "website" much, but we use "web application" quite a bit.

Tweaked this paragraph a bit!